doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -185,129 +166,118 @@ Newly opened modules and sections inherit the current settings.
    arguments ``-set`` and ``-unset`` for setting flags and options
    (cf. :ref:`command-line-options`).
 
-.. _requests-to-the-environment:
-
-Requests to the environment
--------------------------------
+Query commands
+--------------
 
-.. cmd:: Check @term
+Unlike other commands, if a proof is open, query commands may be prefixed with
+:n:`@selector:` to specify which subgoal(s) should be included.  If no selector is provided,
```

Comment:
```suggestion
Unlike other commands, if a proof is open, query commands may be prefixed with
a goal selector (:n:`@num:`) to specify in the context of which goal context, it applies.  If no selector is provided,
```

It's not a syntactic restriction, but query commands only support a single numbered goal selector.

So the "goals" must be replaced by "goal" in the following descriptions.